tools/src/main/script/networktest.sh

```diff
@@ -1,4 +1,8 @@
 #! /bin/bash
-CL_PATH=`find ../ -name '*.jar' | tr "\n" :`
+
+WORKING_DIR=$(dirname "$0")
+cd -P "$WORKING_DIR"/.. > /dev/null
+
+CL_PATH=$(find . -name '*.jar' | tr "\n" :)
 echo "CLASSPATH=$CL_PATH"
-java -classpath $CL_PATH com.navercorp.pinpoint.tools.NetworkAvailabilityChecker ../pinpoint.config
\ No newline at end of file
+java -classpath $CL_PATH com.navercorp.pinpoint.profiler.tools.NetworkAvailabilityChecker pinpoint.config
```

Comment:
The package name of the class is incorrectly included. (pinpoint.~~profiler.~~tools)

please check it again :)